Added detailed documentation on how to start with an ent license (#578)
hazelcast/management-center-docs: docs/modules/getting-started/pages/install.adoc

You can find more details on link:https://hub.docker.com/r/hazelcast/management-center[DockerHub].

> To start Management Center with a valid Enterprise License you can pass the environment variable `MC_LICENSE`:

Reviewer: ```suggestion
To start Management Center with a valid Enterprise Edition license, you can pass the environment variable `MC_LICENSE`:
```

--
====

[[starting-with-enterprise-license]]

> == Starting Management Centre with an enterprise license

Reviewer: ```suggestion
== Starting Management Center with an Enterprise license
```

> If you want to start management centre with a valid enterprise license via command line,

Reviewer: ```suggestion
If you want to start Management Center with a valid Enterprise Edition license via the command line,
```

> you could set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).

Reviewer: ```suggestion
you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```

> you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).

Reviewer 2: ```suggestion
you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```
```suggestion
you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```
```suggestion
you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```
```suggestion
you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```
```suggestion
you can set the `MC_LICENSE` or pass the java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```
```suggestion
you can set the `MC_LICENSE` or pass the Java property `hazelcast.mc.license` environment variable (see xref:deploy-manage:system-properties.adoc#starting-with-a-license[`hazelcast.mc.license` property] for more details).
```

> Alternatively you can set the license via the UI as documented in the xref:deploy-manage:license-management.adoc[] section.

Reviewer: ```suggestion
Alternatively, you can set the license via the UI as documented in the xref:deploy-manage:license-management.adoc[] section.
```

> Alternatively, you can set the license via the UI as documented in the xref:deploy-manage:license-management.adoc[] section.
> 
> [tabs]
> ====
> Linux and Mac::
> +
> --
> [source,bash,subs="attributes+"]
> ----
> export MC_LICENSE=<license-key>
> hz-mc start
> ----
> 
> or
> 
> [source,bash,subs="attributes+"]
> ----
> hz-mc start -Dhazelcast.mc.license=<license key>
> ----
> 
> --
> 
> Windows::
> +
> --
> [source,shell,subs="attributes+"]
> ----
> SET MC_LICENSE=<license-key>
> mc-start.cmd
> ----
> --
> ====

Reviewer 2: ```suggestion
[tabs]
====
Linux and Mac::
+
--
[source,bash,subs="attributes+"]
----
export MC_LICENSE=<license-key>
hz-mc start
----

or

[source,bash,subs="attributes+"]
----
hz-mc start -Dhazelcast.mc.license=<license key>
----

--

Windows::
+
--
[source,shell,subs="attributes+"]
----
SET MC_LICENSE=<license-key>
mc-start.cmd
----
--
====

Alternatively, you can set the license via the UI as documented in the xref:deploy-manage:license-management.adoc[] section.
```

Nit - re-order.
